gapis/api/gles/api/egl.api

```diff
@@ -357,14 +357,12 @@ cmd EGLBoolean eglSurfaceAttrib(EGLDisplay display,
   return ? // TODO
 }
 
-@no_replay
```

Comment:
Please move these API changes into their own commit with a detailed message of why we are making them.